draft-ietf-quic-multipath.md

```diff
@@ -620,6 +620,23 @@ to abandon a path for any reason, for example, removing a hole from
 the sequence of Path IDs in use. This is not an error. The endpoint that
 receive such a PATH_ABANDON frame must treat it as specified in {{path-close}}.
 
+### Closing Duplicate Paths
+
+As noted in {{basic-design-points}}, it is possible to create paths that
+refer to the same 4-tuple. There will be cases where this is intentional,
+for example if the paths use different Differentiated Service markings.
+There may also be cases where this is not intentional, maybe as a result
+of path migration.
```

Comment:
I don't think this unintentional case due to path migration is possible anymore because you either use a new path ID or not. This makes it crystal clear to distinguishes between path migration and opening a new path.

Reply:
Example:

* Client starts path 1 from address 1.1.1.1 to server address 5.5.5.5
* Client starts path 2 from address 2.2.2.2 to server address 5.5.5.5
* both paths are used for a while.
* Server sends packet from address 5.5.5.5 to client address 1.1.1.1, with CID indicating path=2.
* Client receives packet, recognizes a path migration, update source address of path 2 to 1.1.1.1
* Bingo, path 1 and path 2 refer to the same 4 tuple.

Reply:
Example added as part of the implementation section.